fmriprep/workflows/bold/base.py

```diff
@@ -301,17 +298,15 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                    'slicetiming' not in ignore and
                    (_get_series_len(ref_file) > 4 or "TooShort"))
 
-    # Use T2* as target for ME-EPI in co-registration
+    # Check if MEEPI for T2* coregistration target
     if t2s_coreg and not multiecho:
         LOGGER.warning("No multiecho BOLD images found for T2* coregistration. "
                        "Using standard EPI-T1 coregistration.")
         t2s_coreg = False
 
-    # Switch stc off
-    if multiecho and run_stc is True:
-        LOGGER.warning('Slice-timing correction is not available for '
-                       'multiecho BOLD data (not implemented).')
-        run_stc = False
+    # By default, force-bbr for t2s_coreg unless user specifies otherwise
+    if t2s_coreg and (use_bbr is not False):
```

Comment:
How about `if t2s_coreg and use_bbr is None:`? If it's `True`, this branch has no effect anyway, and we can be more explicit that this is the "default" behavior. Also, can you add a note to the docstring (L97-99)?